notebooks/niriss_ami_binary/2_niriss_ami_binary.ipynb

```diff
@@ -11,19 +11,21 @@
    "cell_type": "markdown",
```

Comment:
<p>It looks like there's no need for <code>odir_bp</code> here, so I recommend condensing this cell to <code>datasuperdir = Path('./pipeline_calibrated_data_corr') </code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Done, thanks!

Reply:
Actually this too is used with '+' in many places including nrm_analysis. Reverting to the old path.